src/command.h

```diff
@@ -20,7 +20,7 @@ namespace LAMMPS_NS {
 
 class Command : protected Pointers {
  public:
-  Command(class LAMMPS *lmp) : Pointers(lmp) {};
+  Command(class LAMMPS *lmp) : Pointers(lmp), copymode(0) {};
```

Comment:
@stanmoore1 we both overlooked this one. Fortunately, Coverity Scan reported it after its regular Thursday night check on the develop branch...